src/lib/url.js

```diff
@@ -0,0 +1,64 @@
+const queryParams = ['include', 'fields', 'sort'];
+
+export const parseInclude = include => {
+  return include ? include.split(',') : [];
+};
+
+export const parseFields = query => {
+  let fields = {};
+
+  if (query) {
+    for (let [key, value] of query) {
+      if (key.startsWith('fields') && key.indexOf('[') > -1) {
```

Comment:
When would the indexOf condition be false?

Reply:
I was thinking when there is user input- like a url pasted into the browser bar we'd want to validate that it was a correct filter. This probably doesn't go far enough to really do that. I was mostly adding it since I referred to the indexOf directly below.